app/src/main/java/org/mozilla/fenix/browser/BaseBrowserFragment.kt

```diff
@@ -275,7 +282,11 @@ abstract class BaseBrowserFragment : Fragment(), UserInteractionHandler, Session
                         onCannotOpenFile = {
                             FenixSnackbar.make(view, Snackbar.LENGTH_SHORT)
                                 .setText(context.getString(R.string.mozac_feature_downloads_could_not_open_file))
-                                .setAnchorView(browserToolbarView.view)
+                                .setAnchorView(
+                                    if (view.context.settings().shouldUseBottomToolbar)
+                                        browserToolbarView.view
+                                    else null
```

Comment:
Instead of having to repeat this conditional, what do you think about adding `browserToolbarView#getSnackbarAnchor`?